coriolis/conductor/rpc/server.py

```diff
@@ -1664,6 +1664,21 @@ def report_deployer_failure(
             db_api.set_action_last_execution_status(
                 ctxt, deployment_id, error_status)
 
+    def _normalize_user_scripts(self, user_scripts, instances):
+        """ Removes instance user_scripts if said instance is not one of the
+            selected instances for the replica/migration """
+        if user_scripts is None:
+            user_scripts = {}
+        for instance in list(user_scripts.get('instances', {}).keys()):
+            if instance not in instances:
+                LOG.warn("Removing provided instance '%s' from user_scripts body "
+                         "because it's not included in one of the selected "
+                         "instances for this replica/migration: %s",
+                        instance, instances)
```

Comment:
This will most probably come up if the action runs, but please fix the indentation here:
```suggestion
                LOG.warn("Removing provided instance '%s' from user_scripts body "
                         "because it's not included in one of the selected "
                         "instances for this replica/migration: %s",
                         instance, instances)
```